launchSearch.ts

```diff
@@ -0,0 +1,82 @@
+/*!
+ * Copyright © 2023 United States Government as represented by the
+ * Administrator of the National Aeronautics and Space Administration.
+ * All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import Dockerode from 'dockerode'
+
+const [, , dataDir, logsDir, engine, port, optionsStr] = process.argv
+const options = optionsStr.split(',')
```

Comment:
Rather than converting/splitting/parsing, what about just passing all of the command line arguments as a single JSON-formatting string and `JSON.parse`ing it?